### PR TITLE
Re-release jetson TX2

### PR DESCRIPTION
**Pull request by telphan**
* Morty update
* Switch to gpt 
* Fix partition table
* Fix expander

### CODE REVIEW

File: layers/meta-resin-jetson/recipes-support/tegra-binary-prepare/tegra-binaries-prepare-native_28.1.0.bb [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    cp ${B}/*.encrypt ${DEPLOYDIR}/tegra-binaries-signed
+}
+
+INHIBIT_PACKAGE_STRIP = "1"
+INHIBIT_PACKAGE_DEBUG_SPLIT = "1"
+
+do_configure[depends] += "tegra-binaries:do_preconfigure"
+do_populate_lic[depends] += "tegra-binaries:do_unpack"
+
+addtask do_deploy before do_package afetr do_install
```
Comment (agherzan) [inline layers/meta-resin-jetson/recipes-support/tegra-binary-prepare/tegra-binaries-prepare-native_28.1.0.bb]: "afetr" is not an English word (if intended).

File: layers/meta-resin-jetson/recipes-core/images/resin-image.bbappend [hunk 2]
```diff
@@ -1 +1,3 @@
 include resin-image.inc
+
+IMAGE_FSTYPES_append = " hostapp-ext4"
```
Comment (agherzan) [inline layers/meta-resin-jetson/recipes-core/images/resin-image.bbappend]: It should mention that this still needs update hooks.

File: layers/meta-resin-jetson/recipes-containers/docker/docker_git.bbappend [hunk 3]
```diff
@@ -0,0 +1 @@
+FILES_COMPRESS=""
```
Comment (floion) [inline layers/meta-resin-jetson/recipes-containers/docker/docker_git.bbappend]: upx was fixed for aarch64 and we use this on the artik7 which is aarch64
Are you sure you were using latest meta-resin when you had these problems with it?

File: layers/meta-resin-jetson/recipes-go/resin-provisioner/resin-provisioner_%.bbappend [hunk 4]
```diff
@@ -0,0 +1 @@
+FILES_COMPRESS=""
```
Comment (floion) [inline layers/meta-resin-jetson/recipes-go/resin-provisioner/resin-provisioner_%.bbappend]: upx was fixed for aarch64 and we use this on the artik7 which is aarch64
Are you sure you were using latest meta-resin when you had these problems with it?

File: layers/meta-resin-jetson/recipes-core/images/resin-image.bbappend (hunk 2 above)
Comment (floion) [reply layers/meta-resin-jetson/recipes-core/images/resin-image.bbappend]: @agherzan if this is based on uboot, isn't meta-resin already providing update hooks for u-boot boards?

File: layers/meta-resin-jetson/recipes-containers/docker/docker_git.bbappend (hunk 3 above)
Comment (telphan) [reply layers/meta-resin-jetson/recipes-containers/docker/docker_git.bbappend]: I have used  the latest master and still faced an issue with aarch64.  https://github.com/upx/upx/issues/130

File: layers/meta-resin-jetson/recipes-go/resin-provisioner/resin-provisioner_%.bbappend (hunk 4 above)
Comment (telphan) [reply layers/meta-resin-jetson/recipes-go/resin-provisioner/resin-provisioner_%.bbappend]: I have used  the latest master and still faced an issue with aarch64.  https://github.com/upx/upx/issues/130

File: layers/meta-resin-jetson/recipes-core/images/resin-image.bbappend (hunk 2 above)
Comment (agherzan) [reply layers/meta-resin-jetson/recipes-core/images/resin-image.bbappend]: @floion it had signed binaries written at specific address (this mechanism is not handled by meta-resin).

**Pull request merged**